runtime/src/main/java/io/quarkiverse/ironjacamar/runtime/IronJacamarRecorder.java

```diff
@@ -66,7 +66,8 @@ public Function<SyntheticCreationalContext<CachedConnectionManager>, CachedConne
 
     public void initDefaultBootstrapContext(BeanContainer beanContainer) {
         TransactionIntegration transactionIntegration = beanContainer.beanInstance(TransactionIntegration.class);
-        SecurityIntegration securityIntegration = beanContainer.beanInstance(QuarkusSecurityIntegration.class);
+        SecurityIntegration securityIntegration = beanContainer.beanInstanceFactory(QuarkusSecurityIntegration.class).create()
```

Comment:
I think the easiest way to fix this and still be backward compatible is to register `QuarkusSecurityIntegration` as a CDI bean

Reply:
e.g. make it an `ApplicationScoped`?

Reply:
`@Dependent` is fine for now

Reply:
adding simply `@Dependent` is not enough (at least in `3.5`). I had to make it a synthetic bean.